Code/Framework/AzCore/AzCore/Component/TransformBus.h

```diff
@@ -29,6 +29,13 @@ namespace AZ
     enum class ChildChangeType { Added, Removed };
     using ChildChangedEvent = Event<ChildChangeType, EntityId>;
 
+    //! Used to control the behavior of an entity's transform when its parent's transform changes at runtime.
+    enum class OnParentChangedBehavior : AZ::u8
+    {
+        Update, //!< Update this entity's transform based on the parent's new world transform and this entity's local transform.
```

Comment:
suggestion: Might be worth adding a note that this is the 'default' behavior

Reply:
Good call, will update.